AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.signalrservice/signalr/config.json

```
@@ -9,42 +9,40 @@
     "searchSuffix": "signalr",
     "azureServiceName": "Azure SignalR Services",
     "armApiConfig": {
-		"armApiVersion": "2020-05-01"
-	},
+        "armApiVersion": "2020-05-01"
+    },
     "isSearchEnabled": true,
-	"keystoneDetectorId": "",
+    "keystoneDetectorId": "",
     "liveChatConfig": {
         "isApplicableForLiveChat": false,
         "supportTopicIds": []
     },
     "categories": [
         {
-            "id": "OverallCheck",
-            "name": "Overall Check",
-            "description": "An overall check run to help quickly identify some possible common issues. ",
-            "keywords": [ "Overall Check" ],
-            "color": "rgb(186, 211, 245)",
-            "createFlowForCategory": false,
-            "chatEnabled": false
-        },
-        {
-            "id": "AvailabilityAndPerformance",
+            "id": "AvailabilityAndPerformanceSignalR",
             "name": "Availability And Performance",
             "description": "Investigate issues with your azure signalr service. Find out the common causes and investigate in detail. ",
-            "keywords": [ "Config Sever", "App Health", "Availability" ],
+            "keywords": [ "Config Sever", "App Health", "Availability", "sdk" ],
             "color": "rgb(186, 211, 245)",
             "createFlowForCategory": false,
-            "chatEnabled": false
+            "chatEnabled": false,
+            "overviewDetectorId": "AvailabilityAndPerformanceOverview"
         },
         {
-            "id": "ResourcesUsage",
+            "id": "ResourcesUsageSignalR",
             "name": "Resources Usage",
-            "description": "Are you trying to figure out your resource usage? You can see your SKU plan and your resource usage here. ",
-            "keywords": [ "Resources" ],
+            "description": "Are you trying to figure out your resource usage? You can see your resource usage here. ",

```

Can we reduce the word counts on the description?
```suggestion
            "description": "Find out the resource usage on your SignalR Service.",

```

---

Hi @ruimhu, is it possible to use the description I suggested? I tested the new descriptions you've added, but they don't take up the same number of lines, so the cards don't render as nicely.